src/response.rs

```diff
@@ -125,8 +125,9 @@ impl Response {
     ///
     /// [Read more on MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)
     #[must_use]
-    pub fn set_mime(self, mime: Mime) -> Self {
-        self.set_header(CONTENT_TYPE, mime.to_string())
+    pub fn set_mime(mut self, mime: impl Into<Mime>) -> Self {
+        self.res.set_content_type(mime.into());
+        self
```

Comment:
This line should probably be reverted now?

Reply:
Why?

Reply:
I mean due to #537 and [the discord chaining conversation yesterday](https://discordapp.com/channels/598880689856970762/649056551835009097/714252404111179856).

Reply:
This PR doesn’t change the signature of set_mime, and doing so would be out of its scope. Although Request has changed, Response has not yet as of current master